components/brave_rewards/core/contribution/contribution.cc

```diff
@@ -551,14 +551,9 @@ void Contribution::OnEntrySaved(const std::string& contribution_id,
 
   const std::string& queue_id = queue->id;
 
-  if (wallet_type == constant::kWalletUnBlinded) {
```

Comment:
I think we can now get rid of this wallet type, too (`components/brave_rewards/core/global_constants.h`)?

Reply:
Will remove in https://github.com/brave/brave-browser/issues/36936